internal/updater/verify.go

```diff
@@ -0,0 +1,281 @@
+package updater
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/gopasspw/gopass/pkg/debug"
+	"golang.org/x/crypto/openpgp"
+)
+
+var pubkey = []byte(`
```

Comment:
Ideally we could have a secret key in our CI/DI and its pub key would be here.
I guess this correspond to a local key you have on your machine?

Reply:
Also you could have used a ECDSA key, they are much shorter.

Reply:
Yes, this is my key for now. That's needs to change eventually, but for implementing and testing that should be OK.